alignak/daemon.py

```diff
@@ -720,6 +719,7 @@ def do_daemon_init_and_start(self):
         else:
             self.write_pid()
 
+        self.set_proctitle()
```

Comment:
I should have done this at the very beginning of the `do_daemon_init_and_start` function ...